repos/fdbt-netex-output/src/netex-convertor/point-to-point-tickets/pointToPointTicketNetexHelpers.ts

```diff
@@ -35,6 +35,7 @@ export const getPointToPointScheduledStopPointsList = (fareZones: FareZone[]): S
         version: 'any',
         id: stop.atcoCode ? `atco:${stop.atcoCode}` : `naptan:${stop.naptanCode}`,
         Name: { $t: stop.stopName },
+        NameSuffix: { $t: stop.indicator },
```

Comment:
I think we should be doing something like this 

` NameSuffix: stop.indicator ? { $t: stop.indicator } : null,` 

so that we only create the NameSuffix key if the stop.indicator exists - I think it will create it empty otherwise.

Reply:
From the data I've looked at, it always exisits however if for whatever reason it doesn't at anypoint this is a good catch. Implemented.